cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -117,7 +117,7 @@ macro (build_addon target prefix libs)
                 if("${matchres}" EQUAL 0)
                   string(REPLACE " " ";" loop_var "${loop_var}")
                   list(GET loop_var 1 include_name)
-                  string(REGEX REPLACE "[<>\"]" "" include_name "${include_name}")
+                  string(REGEX REPLACE "[<>\"]|kodi/" "" include_name "${include_name}")
```

Comment:
do we have some constant/variable that contains the app name? I'm just thinking about making it easier for people to rebrand if needed.